pkg/status/templates/header.tmpl

```diff
@@ -160,4 +160,49 @@ Cluster Checks Dispatching
   Status: unknown
 {{- end }}
 {{- end }}
+
+{{- if .admissionWebhook }}
```

Comment:
Have we check that this works on the agent status page (`admissionWebhook` doesn't seems to be populated outside the DCA).

Reply:
Yes it's just like the sections `clusterchecks` and `custommetrics` above in this file.
The template is shared between the agent and the cluster agent, and this can appear on the core agent status only if no DCA in the cluster and leader election is enabled to elect a node agent leader (we don't recommend doing that anymore since the DCA has been GA, but still possible).